backends/p4tools/modules/testgen/targets/pna/dpdk/expr_stepper.cpp

```diff
@@ -18,15 +17,17 @@ PnaDpdkExprStepper::PnaDpdkExprStepper(ExecutionState &state, AbstractSolver &so
 
 // Provides implementations of PNA-DPDK externs.
 // NOLINTNEXTLINE(cppcoreguidelines-interfaces-global-init)
-const ExternMethodImpls PnaDpdkExprStepper::PNA_DPDK_EXTERN_METHOD_IMPLS({});
-
-void PnaDpdkExprStepper::evalExternMethodCall(const IR::MethodCallExpression *call,
-                                              const IR::Expression *receiver, IR::ID name,
-                                              const IR::Vector<IR::Argument> *args,
-                                              ExecutionState &state) {
-    if (!PNA_DPDK_EXTERN_METHOD_IMPLS.exec(call, receiver, name, args, state, result)) {
-        SharedPnaExprStepper::evalExternMethodCall(call, receiver, name, args, state);
+const ExprStepper::ExternMethodImpls<PnaDpdkExprStepper>
+    PnaDpdkExprStepper::PNA_DPDK_EXTERN_METHOD_IMPLS({});
```

Comment:
Should this just be skipped altogether? Once can always add it if necessary in future.

Reply:
I would keep it, makes it a little easier to start adding externs later.

Reply:
On the other hand, it is an extra lookup on the extern invocation path and one would presumably need to look up examples of implementations of other externs anyway.

Reply:
If/def'd the code such that you can easily reenable it once you implement an extern.